src/Workspaces/Core/Portable/Formatting/Rules/BaseIndentationFormattingRule.cs

```diff
@@ -187,7 +187,7 @@ private static TextSpan GetSpanFromTokens(TextSpan span, SyntaxToken token1, Syn
             // current token is inside of the given span, get next token's start position.
             if (token2.Span.End <= span.End)
             {
-                token2 = token2.GetNextToken();
+                token2 = token2.GetNextToken(includeZeroWidth: true);
                 end = token2.Span.Start;
                 if (token2.RawKind == 0)
```

Comment:
@heejaechang I am not entirely sure if this would work because there might be cases where we might need to format the trivia which part of the visible area. Although, I dont have a repro  which explains the above mentioned case